lighthouse-cli/test/smokehouse/test-definitions/pwa/pwa-expectations.js

```diff
@@ -108,7 +108,7 @@ const expectations = [
         // Ignore speed test; just verify that it ran.
         },
         'installable-manifest': {
-          score: 0,
+          score: 1,
```

Comment:
what was the difference here?

Reply:
there's no manifest.  same bug as above.